hbase_storage/hbase1.1.2/src/main/java/com/splicemachine/access/hbase/HBaseConnectionFactory.java

```diff
@@ -65,7 +66,11 @@ private HBaseConnectionFactory(SConfiguration configuration){
         this.namespace=configuration.getNamespace();
         this.namespaceBytes=Bytes.toBytes(namespace);
         try{
-            this.connection=ConnectionFactory.createConnection((Configuration) configuration.getConfigSource().unwrapDelegate());
+            Configuration config = (Configuration) configuration.getConfigSource().unwrapDelegate();
+            this.connection=ConnectionFactory.createConnection(config);
+            Configuration clonedConfig = new Configuration(config);
+            clonedConfig.setInt("hbase.client.retries.number",1);
```

Comment:
Good question.  0 blows chunks immediately and 1 seems to work as advertised.

Regards,
John

> On Aug 11, 2016, at 2:38 PM, Scott Fines <notifications@github.com> wrote:
> 
> In hbase_storage/hbase1.1.2/src/main/java/com/splicemachine/access/hbase/HBaseConnectionFactory.java <https://github.com/splicemachine/spliceengine/pull/70#discussion_r74488789>:
> 
> > @@ -65,7 +66,11 @@ private HBaseConnectionFactory(SConfiguration configuration){
> >          this.namespace=configuration.getNamespace();
> >          this.namespaceBytes=Bytes.toBytes(namespace);
> >          try{
> > -            this.connection=ConnectionFactory.createConnection((Configuration) configuration.getConfigSource().unwrapDelegate());
> > +            Configuration config = (Configuration) configuration.getConfigSource().unwrapDelegate();
> > +            this.connection=ConnectionFactory.createConnection(config);
> > +            Configuration clonedConfig = new Configuration(config);
> > +            clonedConfig.setInt("hbase.client.retries.number",1);
> Should this be 1, or 0? i.e. does it reference only retries, or all tries?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/splicemachine/spliceengine/pull/70/files/fa203b62d27b94d33c5ee07da49a0041c3ab87dc#r74488789>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABw7tVRFANib4D_8chO2EuKfJgYD6otxks5qe3oegaJpZM4JibsW>.
>